### PR TITLE
feat: improve Create Organization flow

### PR DESCRIPTION
**Pull request by acalinica**
## Proposed changes

This PR introduces a new flow for Creating organizations along with some visual changes to the table and a bunch of bug fixes related to Table and infinite data fetching.

## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: packages/apps/spaces/app/organizations/src/components/Actions/Actions.tsx [hunk 1]
```diff
@@ -1,101 +1,60 @@
...
-          onClick={() => {
-            toggleSelection(true);
-            setMode('MERGE');
+    <ButtonGroup size='md' isAttached left='-50%' position='relative'>
+      <Button
+        bg='gray.700'
+        color='white'
+        leftIcon={<Archive />}
+        onClick={onArchiveOrganizations}
+        // borderRight='1px solid'
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organizations/src/components/Actions/Actions.tsx]: 💅  Please remove commented out code

File: packages/apps/spaces/app/organizations/src/components/Columns/Cells/organization/OrganizationCell.tsx [hunk 2]
```diff
@@ -65,7 +67,7 @@ export const OrganizationCell = ({
             {isHovered && (
               <Flex
                 position='absolute'
-                bottom='16px'
+                bottom='8px'
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organizations/src/components/Columns/Cells/organization/OrganizationCell.tsx]: px value in this case should be changed to `2` (prefer theme value over magic numbers whenever possible)

File: packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx [hunk 3]
```diff
@@ -9,23 +11,23 @@ import { Image } from '@ui/media/Image';
...
 
-import { Box } from '@ui/layout/Box';
-import { useLocalStorage } from 'usehooks-ts';
 import { useJune } from 'app/src/hooks/useJune';
 import { LogOut01 } from '@ui/media/icons/LogOut01';
 import { getGraphQLClient } from '@shared/util/getGraphQLClient';
+import { useOrganizationsMeta } from '@shared/state/OrganizationsMeta.atom';
 import { useGlobalCacheQuery } from '@shared/graphql/global_Cache.generated';
 
+import logo from './customer-os.png';
```
Comment (Kasia9090) [inline packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx:19]: this could move it to public folder

**Review by Kasia9090 — CHANGES_REQUESTED** [review]

File: packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx (hunk 3 above)
Comment (acalinica) [reply packages/apps/spaces/app/src/components/RootSidenav/RootSidenav.tsx:19]: About this, I think it loads better and has blur placeholder if we load it this way.

**Review by Kasia9090 — APPROVED** [review]

**Pull request merged**